### DISCUSSION TITLE
Found JP Channel Streaming But...

### DISCUSSION THREAD

**Discussion by Central302** [iptv-org/community#176] [Show and tell]
Looks Like Someone's Restreaming Server?
http://123.253.32.125:9995/ubtv/show_headers

**Comment by BellezaEmporium** [comment]
> Looks Like Someone's Restreaming Server? http://123.253.32.125:9995/ubtv/show_headers

Sounds like you've stumbled upon an restreaming host indeed. Interesting the fact that they point out "streamlink links" as a resource.

What's even weirder is, if you look at the "Streamlink" resource, it points out to a host with all sorts of tokens and IDs. Holy grail you might have found lol. Tested a few Chinese + Japanese channels and they work perfectly.

**Comment by BellezaEmporium** [comment]
c20193 -> TV Tokyo
c20200 -> Nihon TV G+
c20194 -> Fuji TV
